docs/developer-docs/integrations/bitcoin/bitcoin-how-it-works.md

```diff
@@ -2,105 +2,93 @@
-- A chain-key signatures based on a novel threshold ECDSA protocol.
 
-This page provides a general overview of the Bitcoin integration technology. For a deeper technical explanation of ckBTC, please see the [wiki page](https://wiki.internetcomputer.org/wiki/Chain-key_Bitcoin).
+The Internet Computer Protocol has been integrated with the Bitcoin network to obtain a direct technical integration between the two. 
 
-## Protocol-level integration of ICP with the Bitcoin network
-
-Through the protocol-level integration of ICP with the Bitcoin network, ICP can obtain Bitcoin blocks directly from the Bitcoin network and process the contained transactions. This integration makes it possible to maintain the full Bitcoin UTXO set on-chain on ICP. Canisters can run queries against the full Bitcoin UTXO set. This allows canisters to know about the held UTXOs, and thus balance, of any Bitcoin address, including their own addresses.
-
-## Chain-key ECDSA signatures
-
-Canisters themselves can have ECDSA keys using a novel chain-key ECDSA signature protocol suite for threshold ECDSA, and so can receive and hold Bitcoin. Canisters also can create Bitcoin transactions and submit them via the Bitcoin API to the Bitcoin network. They use the chain-key ECDSA functionality to request threshold ECDSA signatures to spend UTXOs in a transaction to be submitted to the Bitcoin network. Chain-key ECDSA signatures are a new member of ICP's chain-key cryptography toolbox of protocols. Chain-key ECDSA signatures are much more than just a threshold ECDSA implementation as they for example also comprise protocols for secure distributed key generation and key rotation, which are crucial from a systems perspective to make threshold signing secure and practically viable. Details regarding ICP's chain-key ECDSA signature protocol can be found on the corresponding documentation page [here](../t-ecdsa/t-ecdsa-how-it-works.md).
-
-The protocol-level Bitcoin integration and chain-key ECDSA signature protocols each expose an API on the management canister. Those APIs are the system-level APIs engineers use to write Bitcoin smart contracts on ICP. The APIs are low-level APIs designed around the concepts of Bitcoin UTXOs and transactions and are non-trivial to use and require an in-depth understanding of how Bitcoin works. The chain-key ECDSA signature API is also generically useful for any ECDSA use case, e.g., integration with other blockchains such as Ethereum.
+![Bitcoin Integration](../_attachments/bitcoin_integration.png)
 
-This page gives a high-level overview of the above mentioned technology behind the direct Bitcoin integration. For details, it is recommended to refer the reader to the [Bitcoin page on the Internet Computer wiki](https://wiki.internetcomputer.org/wiki/Bitcoin_integration) as well as the [threshold ECDSA documentation page](../t-ecdsa/t-ecdsa-how-it-works.md).
+The basis of this integration includes:
```

Comment:
I think this list contains too many new terms and may overwhelm the reader (UTXO, chain-key signature, ECDSA, managment canister). The reader who doesn't know details of Bitcoin and ICP will be lost here.

Let's try to find a more gentle explanation of how Bitcoin integration works at the high-level assuming that the reader is not an expert in Bitcoin and ICP. I would also go with informal and less academic language.

For example:
"Smart contracts running on IC can directly interact with the Bitcoin network without any bridges." (continuing the previous comment)
"This is possible because IC nodes also participate in the Bitcoin network. They fetch and validate the Bitcoin blocks and make them available for smart contracts to query and read."
"Being able to read the Bitcoin state fully on-chain opens many interesting use cases. However, for the full integration, we also need to allow smart contracts on IC to submit Bitcoin transactions. This is possible thanks to the unique feature of IC that allows to have decentralized private keys for signing Bitcoin transactions. You can read more about how it works in this blog post: https://medium.com/dfinity/how-icps-bitcoin-integration-secures-private-keys-c2af14597846"

Reply:
I like the idea of further simplifying, I'd just be careful with this particular sentence

> This is possible because IC nodes also participate in the Bitcoin network

because it's not really true (or it depends on what level of abstraction you're looking at things). IC nodes are not full bitcoin nodes as one might think if they read "they participate in the Bitcoin network".